go.mod

```diff
@@ -2,7 +2,10 @@ module github.com/vmware-tanzu/tanzu-cli
 
 go 1.19
 
-replace cloud.google.com/go => cloud.google.com/go v0.102.1
+replace (
+	cloud.google.com/go => cloud.google.com/go v0.102.1
+	github.com/vmware-tanzu/tanzu-plugin-runtime => github.com/anujc25/tanzu-plugin-runtime v0.0.0-20230220221618-695e5b2b7d7b
```

Comment:
Update this once https://github.com/vmware-tanzu/tanzu-plugin-runtime/pull/17 is merged.